Header header

geometry_msgs/Pose[] poses
uint32[] mnIds